bin/apisix

```diff
@@ -290,13 +290,23 @@ http {
     server {
         {%if https_admin then%}
         listen {* port_admin *} ssl;
+
+        {%if mtls and mtls.mtls_enable then%}
+        ssl_verify_client on;
+        ssl_certificate      {* mtls.server_cert *};
+        ssl_certificate_key  {* mtls.server_key *};
+        ssl_client_certificate {* mtls.ca_cert *};
```

Comment:
`ssl_client_certificate` is used to verify client certificates, do we need it?

Reply:
yes, `ssl_verify_client` depends on  `ssl_client_certificate` to verify client requests.